site/profile/manifests/windows/baseline.pp

```diff
@@ -1,14 +1,13 @@
-class profile::windows::baseline {
+class profile::baseline {
 
-  class { 'chocolatey':
-    notify => Reboot['afterchocolatey'],
+  exec { 'rename-admin':
```

Comment:
I thought about this a bit more, and a separate baseline makes sense actually. Means we can have functionally separate demos instead of a single stack. Also means we can keep in the PS example too without it interfering with your stuff.